lldb/source/Plugins/Language/CPlusPlus/CPlusPlusLanguage.cpp

```diff
@@ -1636,6 +1622,36 @@ static void LoadCommonStlFormatters(lldb::TypeCategoryImplSP cpp_category_sp) {
             return LibStdcppStringSummaryProvider(valobj, stream, options);
           },
           "MSVC STL/libstdc++ std::wstring summary provider"));
+
+  auto smart_ptr_creator =
```

Comment:
These registrations are getting a bit noisy in my opinion. Could we move all this to a separate helper called `RegisterCommonStlSmartPtrFormatters` (or something like that?